versioned_docs/version-2.7/reference-guides/best-practices/rancher-managed-clusters/tips-to-set-up-containers.md

```diff
@@ -48,9 +52,10 @@ If you set a resource limit for the namespace that the pod is deployed in, and t
 
 :::
 
-It is recommended to define resource requirements on the container level because otherwise, the scheduler makes assumptions that will likely not be helpful to your application when the cluster experiences load.
+As a starting point, we recommend setting your CPU request at 50m and memory request at 100Mi. You should choose your limits based on the resources available in your environment and your use case with Rancher. It is recommended to define resource requirements on the container level because otherwise, the scheduler makes assumptions that will likely not be helpful to your application when the cluster experiences load.
```

Comment:
This page seems to be about general containers, but the intention is to explain about the specific Pod Rancher automatically installs to Rancher registered clusters (cattle-cluster-agent).

See also https://jira.suse.com/browse/SURE-7353?focusedId=1344617&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-1344617

Reply:
Ah ok, I'll make some adjustments. Thanks Silvio.

Reply:
I've reworded it, as suggested above: 

_As a base line we recommend setting the CPU request at 50m and memory request at 100Mi. However, it is important that you ensure your cluster has the correct resources available and the use case has been assessed appropriately before doing so._

Reply:
@asettle my remark was more about the position of this remark in the documentation. SURE-7353 is not about any general container, it is specific to the cattle-cluster-agent pod Rancher installs in downstream clusters to manage them.

Is the intention to move this is some place more specific? This page seems to be about general recommendations for any containerized workload?

Reply:
@moio I moved the info to the [dedicated Rancher agent page](https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/launch-kubernetes-with-rancher/about-rancher-agents) that we have. Let me know if that's better.